{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "MQ Appliance",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "9.1.LTS"
                        },
                        {
                            "status": "affected",
                            "version": "9.1.CD"
                        },
                        {
                            "status": "affected",
                            "version": "9.2.LTS"
                        },
                        {
                            "status": "affected",
                            "version": "9.2.CD"
                        }
                    ]
                }
            ],
            "datePublic": "2021-11-05T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM MQ 9.1 LTS, 9.1 CD, 9.2 LTS, and 9.2CD is vulnerable to a denial of service attack caused by an issue processing message properties. IBM X-Force ID: 205203."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "HIGH",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 5.3,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 4.6,
                        "temporalSeverity": "MEDIUM",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/I:N/C:N/AC:H/S:U/A:H/UI:N/PR:L/AV:N/RC:C/RL:O/E:U",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-11-08T16:50:32",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6513681"
                },
                {
                    "name": "ibm-mq-cve202129843-dos (205203)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/205203"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2021-29843",
        "dateReserved": "2021-03-31T00:00:00",
        "dateUpdated": "2021-11-08T16:50:32",
        "state": "PUBLISHED",
        "datePublished": "2021-11-08T16:50:32.227400Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
